modules/uhsg_user_sync/src/SamlAuth/UserSyncSubscriber.php

```diff
@@ -410,44 +393,72 @@ protected function setTechnicalDegreeProgrammes(SamlAuthUserSyncEvent $event, $s
 
     // Use Sisu Service
     // Map StudentDegreeProgram to a known degree programme and create flagging
-    if($use_sisu_service && $studyrights = $this->studyRightsService->getActiveStudyRights($student_number)) {
-      $technical_condition_field_name = $this->config->get('technical_condition_field_name');
-      $primary_field_name = $this->config->get('primary_field_name');
-
-      // Loop trough all studyrights
-      foreach($studyrights as $studyright) {
-        // If code matches our degree_program code then proceed
-        if (isset($known_degree_programmes[$studyright->getCode()])) {
+    if($use_sisu_service){
+      if ($studyrights = $this->studyRightsService->getActiveStudyRights($student_number)) {
+        $technical_condition_field_name = $this->config->get('technical_condition_field_name');
+        $primary_field_name = $this->config->get('primary_field_name');
+        $studyright_codes = array();
```

Comment:
This isn't used in anywhere?

Reply:
True that. I now removed this outdated variable that is no longer used.